src/write.c

```diff
@@ -449,6 +450,7 @@ avifEncoder * avifEncoderCreate(void)
     encoder->scalingMode = noScaling;
     encoder->data = avifEncoderDataCreate();
     encoder->csOptions = avifCodecSpecificOptionsCreate();
+    encoder->headerFormat = AVIF_HEADER_FULL;
```

Comment:
Nit/optional: Move this right before `return encoder;` (line 458) so that this doesn't separate the allocations from the checks for allocation failures.